streaming/src/main/scala/org/apache/spark/streaming/ui/AllBatchesTable.scala

```diff
@@ -26,8 +26,10 @@ private[ui] abstract class BatchTableBase(tableId: String) {
   protected def columns: Seq[Node] = {
     <th>Batch Time</th>
       <th>Input Size</th>
-      <th>Scheduling Delay</th>
-      <th>Processing Time</th>
+      <th>Scheduling Delay
+        {SparkUIUtils.tooltip("Time taken by Streaming to schedule a batch", "top")}</th>
```

Comment:
I think he wanted ot highlight that the delay is in the streaming scheduler and not the DAGScheduler/Taskscheduler. How about "Time taken by streaming scheduler to submit jobs of a batch."